schematic/models/validate_attribute.py

```diff
@@ -2098,7 +2110,7 @@ def filename_validation(
             vr_errors, vr_warnings = GenerateError.generate_filename_error(
                 val_rule=val_rule,
                 attribute_name="Filename",
-                row_num=str(index),
+                row_num=str(index + 2),
```

Comment:
May I ask why the +2 here?

Reply:
This is how the row number indexing is done everywhere else in `validate_attribute.py`. [Example](https://github.com/Sage-Bionetworks/schematic/blob/045d8c2004f7615778f47f9be3b931de95d012cc/schematic/models/validate_attribute.py#L959)

So I did this to make it consistent and to avoid confusion like [this](https://sagebionetworks.slack.com/archives/C07HLTBS5D2/p1726156665980609?thread_ts=1726000362.049489&cid=C07HLTBS5D2).

Reply:
May want to add a comment about that in the code